Library/Formula/nspr.rb

```diff
@@ -32,12 +34,12 @@ def install
         --enable-strip
         --enable-pthreads
         --enable-ipv6
-        --enable-macos-target=#{MacOS.version}
       ]
-      args << "--enable-64bit" if MacOS.prefer_64_bit?
+      args << "--enable-macos-target=#{MacOS.version}" if OS.mac?
+      args << "--enable-64bit" if MacOS.prefer_64_bit? || OS.linux?
```

Comment:
Sorry it took so long to respond, I've been sick the last few days. Will try to get to this later today.